File: pajton/suma_parzyste.py

#Program sumuje liczby parzyste w zakresie <0,100>
def main(args):
    start=int(input("Podaj liczbę poczontkowom: "))
    koniec=int(input("Podaj liczbę końcową: "))
    suma = 0
    for liczba in range (start,koniec+1):
        if liczba%2==0:
            suma = suma + liczba
    print(suma)
 
 
 
 
 
 
 
 
 
    return 0











if __name__ == '__main__':
    import sys
    sys.exit(main(sys.argv))

File: cpp/suma_parzyste.cpp



#include <iostream>

using namespace std;

int main(int argc, char **argv)
{
	int suma, liczba;
	int start, koniec;
	
	suma = 0;
	start=koniec=0;
	
	cout << "Podaj liczbę początkową: ";
	cin >> start;
	cout << "Podaj liczbę końcową: ";
	cin >> koniec;
	
	
	for (liczba = start; liczba < koniec+1; liczba++){
		if(liczba%2==0)
		suma=suma+liczba;
	}
	cout << suma;
	
	
	
	
	
	return 0;
}

